io.github.accessory.minus_games_client.yml

```diff
@@ -0,0 +1,445 @@
+    directory: lib/i386-linux-gnu/GL
+    version: *gl-version
+    versions: *gl-versions
+    subdirectories: true
+    no-autodownload: true
+    autodelete: false
+    add-ld-path: lib
+    merge-dirs: *gl-merge-dirs
+    download-if: active-gl-driver
+    enable-if: active-gl-driver
+    autoprune-unless: active-gl-driver
+
+  org.freedesktop.Platform.GL32.Debug:
+    directory: lib/debug/lib/i386-linux-gnu/GL
+    version: *gl-version
+    versions: *gl-versions
+    subdirectories: true
+    no-autodownload: true
+    merge-dirs: *gl-merge-dirs
+    enable-if: active-gl-driver
+    autoprune-unless: active-gl-driver
+
+  org.freedesktop.Platform.VAAPI.Intel.i386:
+    directory: lib/i386-linux-gnu/dri/intel-vaapi-driver
+    version: *runtime-version
+    versions: *runtime-version
+    autodelete: false
+    no-autodownload: true
+    add-ld-path: lib
+    download-if: have-intel-gpu
+    autoprune-unless: have-intel-gpu
+
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: true
+    autodelete: false
+
+  org.freedesktop.Platform.ffmpeg_full.i386:
+    directory: lib32/ffmpeg
+    add-ld-path: .
+    version: *runtime-version
+    no-autodownload: true
+    autodelete: false
+
+  com.valvesoftware.Steam.CompatibilityTool:
+    subdirectories: true
+    directory: share/steam/compatibilitytools.d
+    version: stable
+    versions: stable;beta;test
+    no-autodownload: true
+    autodelete: true
+
+  com.valvesoftware.Steam.Utility:
+    subdirectories: true
+    directory: utils
+    version: stable
+    versions: stable;beta;test
+    add-ld-path: lib
+    merge-dirs: bin;lib/python3.10/site-packages;share/vulkan/explicit_layer.d;share/vulkan/implicit_layer.d;share/steam/compatibilitytools.d;
+    no-autodownload: true
+    autodelete: true
+
+modules:
+  - name: minus_games_client
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/rust-stable/bin
+      env:
+        "CARGO_HOME" : "/run/build/minus_games_client/cargo"
+    build-commands:
+      - cargo --offline fetch --manifest-path Cargo.toml --verbose
+      - cargo --offline build --release --verbose --bin minus_games_client
+      - install -Dm755 target/release/minus_games_client /app/bin/minus_games_client
+      - install -Dm644 other/assets/client/io.github.accessory.minus_games_client.svg /app/share/icons/hicolor/scalable/apps/io.github.accessory.minus_games_client.svg
+      - install -Dm644 other/assets/client/io.github.accessory.minus_games_client.desktop /app/share/applications/io.github.accessory.minus_games_client.desktop
+      - install -Dm644 other/assets/client/io.github.accessory.minus_games_client.metainfo.xml /app/share/metainfo/io.github.accessory.minus_games_client.metainfo.xml
+    sources:
+      - type: git
+#        path: ../..
+        url: https://github.com/Accessory/minus_games/
+        branch: main
+      - type: dir
```

Comment:
do not use dir source. also why do you need it?

Reply:
removed